.dependabot/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+    default_reviewers:
+      - "karelhala"
```

Comment:
Do you want to change this to me?

Reply:
Yah, I keep copying that from one place where I am the default reviewer